config/locales/da.yml

```diff
@@ -9,6 +9,7 @@ da:
         admin:
           users:
             delete: Slet bruger
+              message: Er du sikker på du vil slette brugeren?
```

Comment:
this indentation seems out, or perhaps you meant to nest it under the other delete? https://github.com/refinery/refinerycms-authentication-devise/blob/master/config/locales/en.yml#L57